src/main/java/module/lineup/AustellungSpielerTable.java

```diff
@@ -13,9 +14,11 @@
 import core.gui.model.UserColumnFactory;
 import core.model.HOVerwaltung;
 import core.model.UserParameter;
+import core.model.player.IMatchRoleID;
 import core.model.player.Player;
 import core.util.Helper;
 import module.playerOverview.PlayerTable;
+import module.playerOverview.SpielerUebersichtsPanel;
```

Comment:
I think this import is unused.

Reply:
correct I removed it in new commit along some more code cleaning in this file